html/gui/js/modules/Usage.js

```diff
@@ -659,6 +659,9 @@ Ext.extend(XDMoD.Module.Usage, XDMoD.PortalModule, {
                                 CCR.xdmod.ui.actionLogout.defer(1000);
 
                             }
+                            if (resp.success === false) {
+                                Ext.MessageBox.alert("Error", resp.message);
```

Comment:
I don't even remember this being a rule, but single quotes are required (https://travis-ci.org/github/ubccr/xdmod/jobs/670845844#L1333):

```suggestion
                                Ext.MessageBox.alert('Error', resp.message);
```

Reply:
copy and paste from a few lines above failed me...